test/fixtures/eslint-with-deprecated-rules.json

```diff
@@ -1,14 +1,17 @@
 {
   "plugins": [
     "plugin",
-    "@scope/scoped-plugin"
+    "@scope/scoped-plugin",
+    "@scope"
   ],
   "rules": {
     "foo-rule": [2],
     "old-rule": [2],
     "plugin/foo-rule": [2],
     "plugin/old-plugin-rule": [2],
     "scoped-plugin/foo-rule": [2],
-    "scoped-plugin/old-plugin-rule": [2]
+    "scoped-plugin/old-plugin-rule": [2],
```

Comment:
this seems wrong to me; it should be `@scoped-plugin/`, not `scoped-plugin/`?

Reply:
That's what docs for `typescript-eslint` say - https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#usage

Reply:
Those docs seem like they include the `@` in the rule name to me - are we looking at a different link?

Reply:
@DylanVann - typescript-eslint is a different case. it uses the special case name of `@typescript-eslint/eslint-plugin` which is automatically handled by eslint such that `/eslint-plugin` is not required anywhere.

However in this case, this looks like the old test cases were wrong.
Looking at an example of a package that doesn't use special naming (https://github.com/getify/eslint-plugin-proper-arrows#eslintrcjson), it should instead be `@scope/scoped-plugin/old-plugin-rule`

Reply:
Right - the only thing that makes anything a scope is the leading @, ever